src/main/java/teammates/ui/controller/InstructorFeedbackEditCopyPageData.java

```diff
@@ -7,10 +7,30 @@
 
 public class InstructorFeedbackEditCopyPageData extends PageData {
     public List<CourseAttributes> courses;
-    public String courseId;
-    public String fsName;
+    private String courseId;
+    private String fsName;
     
     public InstructorFeedbackEditCopyPageData(AccountAttributes account) {
         super(account);
     }
+
+    public List<CourseAttributes> getCourses() {
+        return courses;
+    }
+
+    public String getCourseId() {
+        return courseId;
+    }
+
+    public String getFsName() {
+        return fsName;
+    }
+
+    public void setCourseId(String courseId) {
+        this.courseId = courseId;
+    }
+
+    public void setFsName(String fsName) {
```

Comment:
Maybe just set these variables in the constructor?
The constructor can also be overloaded if you need to include the courses in some situations.